src/hardware/vga_dac.cpp

```diff
@@ -305,8 +305,11 @@ void VGA_DAC_CombineColor(const uint8_t palette_idx, const uint8_t color_idx)
 {
 	vga.dac.combine[palette_idx] = color_idx;
 
-	if (vga.mode != M_LIN8) {
-		// Used by copper demo; almost no video card seems to support it
+	// Mimic the legacy `machine = vgaonly` behaviour when emulating the
```

Comment:
Removed the Copper demo comment as it seems to be outdated. The demo runs fine with `svga_paradise` and I haven't noticed any wrong colours.